### PR TITLE
Fix centroid calculation. Bump turf to v6.5

### PR DESCRIPTION
**Pull request by birkskyum**
Update turf to v6.5.

This fixes a bug where the centroid calculation erroneously would include the first point of a polygon twice, thus giving it double weight. Single- and Multi-polygon tests are updates

[Changelog](https://github.com/Turfjs/turf/blob/master/CHANGELOG.md#700)

### CODE REVIEW

File: draftlogs/7115_change.md [hunk 1]
```diff
@@ -0,0 +1,2 @@
+ - Update turf.js to v7 [[#7115](https://github.com/plotly/plotly.js/pull/7115)]
```
Comment (archmoj) [inline draftlogs/7115_change.md:1]: Let's rename the filename and log as a bug fix.

Comment (birkskyum) [reply draftlogs/7115_change.md:1]: There is bugs in the multi polygon test as well, that I'm investigating

Comment (birkskyum) [reply draftlogs/7115_change.md:1]: I have updated the draftlog

**Comment by archmoj** [comment]
Please update the PR title and description to mention the bug fix.
Thank you.

File: draftlogs/7115_fix.md [hunk 2]
```diff
@@ -0,0 +1,2 @@
+ - Update turf.js to v7 [[#7115](https://github.com/plotly/plotly.js/pull/7115)]
+
```
Comment (archmoj) [inline draftlogs/7115_fix.md:2]: Please add a thank you message to your github profile similar to those found in the changelog.
Merci beaucoup!

File: src/lib/geo_location_utils.js [hunk 3]
```diff
@@ -2,9 +2,9 @@
 
 var d3 = require('@plotly/d3');
 var countryRegex = require('country-regex');
-var turfArea = require('@turf/area');
-var turfCentroid = require('@turf/centroid');
-var turfBbox = require('@turf/bbox');
+var { area: turfArea } = require('@turf/area');
```
[the remark this answers is not in the document]
Comment (birkskyum) [reply src/lib/geo_location_utils.js:5]: they are turned into this, so it's a standard [destructuring assignment](https://caniuse.com/?search=%20destructuring%20assignment):

```
var {
  area: turfArea
} = __webpack_require__(9532);
```

**Comment by birkskyum** [comment]
The bugfix is in v7, I just updated the tests to match it

**Comment by birkskyum** [comment]
It's rolled back to 6.5 now with the bugfix

**Comment by birkskyum** [comment]
It's CI Green now (except for usual suspects)

**Comment by archmoj** [comment]
Please open your new PR for v7.
Thank you.

**Pull request merged**